app/dtos.py

```diff
@@ -0,0 +1,10 @@
+from datetime import datetime
+
+
+class UserDTO:
+    def __init__(self, name: str, email: str, password: str, id: int = None, created_date: datetime = None):
+        self.name = name
+        self.email = email
+        self.password = password
+        self.id = id
```

Comment:
dałbyś id jako 1 parametr?

Reply:
@SmallCelestial  da się to inaczej jakoś obrobić? chodzi o to zęby konstruktor nie przyjmował parametru id, ale też żeby był na piewrszmy miejscu w konstruktorze i jako pole

Reply:
```python 
class UserDTO:
    def __init__(self,  id: int = None, name: str = "", email: str = "", password: str = "", created_date: datetime = None):
        self.id = id
        self.name = name
        self.email = email
        self.password = password
        self.created_date = created_date
```

Reply:
Poprawiłem